source/ADIOS.cpp

```diff
@@ -25,10 +25,8 @@
 #include "engine/bp/BPFileReader.h"
 #include "engine/bp/BPFileWriter.h"
 
-#ifdef ADIOS_HAVE_DATAMAN // external dependencies
 #include "engine/dataman/DataManReader.h"
```

Comment:
OK. I got it now. Sure. That makes perfect sense. So what I need to do is just reverse ADIOS.CPP to how it looked like before?